microsoft-edge/progressive-web-apps-chromium/how-to/microsoft-store.md

```diff
@@ -60,20 +60,27 @@ Now that you have your app publishing information, generate a Windows app packag
 To generate an app package:
 
 1.  Go to [PWA Builder](https://www.pwabuilder.com).
-1.  Type the URL of your PWA and click **Start**.
-1.  Once your report is complete, make sure your PWA is store-ready. If your PWA score is too low, you can visit the **Manifest Options** and **Service Worker Options** and check out the sections that need work.
-1.  When your PWA is ready to be published, click **Next**, and then select the **Store Package** button in the **Windows** section of the publish page.
-1.  Paste the following values, which you saved in the [Create an app reservation](#create-an-app-reservation) section:
-    *   **Package ID**
-    *   **Publisher ID**
-    *   **Publisher Display Name**
+
+1.  Under **Ship your PWA to app stores**, enter the URL of your PWA in the text box, and then click the **Start** button.
+
+    Your PWA is evaluated to ensure it is store-ready.
+
+1.  If the evaluation in the PWABuilder report card page indicates that your PWA is ready to be packaged, continue to the next step.  If the evaluation indicates that your PWA is not ready to be packaged, examine  the **Action Items** field to see how to improve your PWA.
+
+1.  Once your PWA is ready to be packaged, in the upper-right corner of the PWABuilder report card page, click the **Package For Stores** button.
```

Comment:
Is the button label:
*  Package For Stores
*  Package for Stores
*  Package for stores

@zateutsch

Reply:
Package For Stores.  Added screenshot.